README.md

```diff
@@ -5,7 +5,7 @@
 
 µRaiden is an off-chain, cheap, scalable and low-latency micropayment solution.
 
-[µRaiden documentation](http://microraiden.readthedocs.io/en/latest/)
+[µRaiden documentation](https://microraiden.readthedocs.io/en/docs-develop/)
```

Comment:
Wouldn't it be better to just have `https://microraiden.readthedocs.io` ? This is the main docs link and now it redirects to `docs-develop`. We can make it redirect to anything if needed, without changing the readme again.